tools/setup.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python3
+
+# Copyright (C) 2018-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+"""
+Use this script to create a wheel with OpenVINO™ Python* tools:
+
+$ python setup.py sdist bdist_wheel
+"""
+
+from setuptools import setup, find_packages
+
+with open("README.md", "r", encoding="utf-8") as f:
+    long_description = f.read()
+
+with open('benchmark/requirements.txt') as f:
+    required = f.read().splitlines()
+
+NAMESPACE = 'openvino.tools'
+packages = find_packages()
```

Comment:
please rename this variable to tools_packages to mitigate misunderstanding in the #43 line